src/dlls/mscordac/CMakeLists.txt

```diff
@@ -184,4 +184,21 @@ endif(WIN32)
 target_link_libraries(mscordaccore PRIVATE ${COREDAC_LIBRARIES})
 
 # add the install targets
-install_clr(mscordaccore)
+install_clr(TARGETS mscordaccore)
+install_clr(TARGETS mscordaccore DESTINATION sharedFramework)
+if(WIN32)
```

Comment:
How was the logic below previously handled? Is this a duplication from where else?

Reply:
This logic is copied from https://github.com/dotnet/coreclr/blob/012c53ba4644ace6997dec0d608b5fd00a77983c/src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/runtime.Windows_NT.Microsoft.NETCore.Runtime.CoreCLR.props#L40-L46 which will go away when we move to live-live builds in the consolidated repo.

Reply:
Can you please create an issue or add it to an existing issue so these kind of changes are removed during that transition?

Reply:
It's tracked as part of moving to live-live builds and removing the Microsoft.NETCore.Runtime.CoreCLR package.